src/exo/LoopIR_compiler.py

```diff
@@ -375,11 +375,10 @@ def from_lines(x):
     public_fwd_decls = []
 
     # Body contents
-    memory_code = _compile_memories(find_all_mems(proc_list))
-    builtin_code = _compile_builtins(find_all_builtins(proc_list))
     private_fwd_decls = []
     proc_bodies = []
     instrs_global = []
+    new_proc_list = []
```

Comment:
I found this name confusing. Maybe you can call it non_instr_procs. Or at least add a comment after it.

Reply:
`analyzed_proc_list` ?